docs/en/05_Releases_and_changelogs/cwp_2.6.0.md

```diff
@@ -8,6 +8,35 @@ This release includes CMS and Framework version 4.6.0.
 
 Upgrading to Recipe 2.6.0 is recommended for all CWP sites. This upgrade can be carried out by any development team familiar with SilverStripe. However, if you would like SilverStripe's assistance, you can request support via the [Service Desk](https://www.cwp.govt.nz/service-desk/new-request/).
 
+### Mimevalidator is now a part of silverstripe/recipe-core
+
+For most CWP installations this should be a transparent change. However, if you changed the mimevalidator configuration,
+a manual step is required when upgrading to CWP 2.6.0.
+
+As a result of [RFC 8782](https://github.com/silverstripe/silverstripe-framework/issues/8782),
+the module `silverstripe/mimevalidator` is now included into `silverstripe/recipe-core`,
+which is a part of `cwp/cwp-recipe-core`.
+
+Historically, `silverstripe/mimevalidator` was a dependency of `cwp/cwp-recipe-core < 2.6` and
+`cwp/cwp < 2.6` had a configuration for that module (at [vendor/cwp/cwp/_config/mimevalidator.yml](https://github.com/silverstripe/cwp/blob/2.5.2/_config/mimevalidator.yml)).
+
+After the upgrade, your project will get a new configuration file at [app/_config/mimevalidator.yml](https://github.com/silverstripe/recipe-core/blob/4.6.0/app/_config/mimevalidator.yml),
+which is compatible with the preexisting configuration file. However, if you introduced any custom
+settings for mimevalidator, the new file may override it by accident. We advise to either
+move your configuration into the new file, or delete the new file so it does not interfere.
+
+If you do not have any custom settings for the mimevalidator module, you need not do anything.
+
+If you opted-out of `silverstripe/mimevalidator` module by manually removing it from your composer.json,
+you should consider disabling it through the configuration, since it will be automatically installed on the next upgrade.
```

Comment:
```suggestion
If you have opted-out of `silverstripe/mimevalidator` module by removing it from your composer.json,
you should consider disabling it via configuration, since it will be automatically reinstalled when upgrading to 2.6.0.
```